module-1/lab-functional-programming/your-code/BarOfWords.ipynb

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
1. You should use an array of paths. 
```Python
#Something like this using relative paths:
docs = ['./doc1.txt', './doc2.txt', './doc3.txt']
````
2. When you make a pull request please send the notebook with the execution. 
3. This statement doesn't throw the requested output 
```Python 
print("Los unos y ceros, aunque coinciden en frecuencia con la respuesta propuesta, \nestán en distinto orden porque el orden de mi bag_of_words es distinto, \npero cumplen con el objetivo: corresponden a la apareción o no de cada palabra. \n")
print(term_freq)
print('\nbag of words: ', bag_of_words)

#The output should be 
[[1, 1, 1, 0, 0, 0, 0, 0, 0], [1, 0, 0, 1, 1, 0, 0, 0, 0], [1, 0, 0, 1, 0, 1, 1, 1, 1]]
```
4. Good try in the bag of words. You should keep your code as simple as possible and must be readable for everyone you need to share it with.